test/e2e/default-fixture.js

```diff
@@ -1,7 +1,11 @@
-const { mockNetworkState } = require('../stub/networks');
+const { mockNetworkStateOld } = require('../stub/networks');
 const { CHAIN_IDS } = require('../../shared/constants/network');
 const { FirstTimeFlowType } = require('../../shared/constants/onboarding');
 
+// TODO: Should we bump this?
+// The e2e tests currently configure state in the schema of migration 74.
+// This requires us to specify network state in the old schema, so it can run through the migrations.
+// We could bump this to latest, but it breaks too many other things to handle right now.
 const FIXTURE_STATE_METADATA_VERSION = 74;
```

Comment:
ughhhh yeah... This tech debt has been lingering for awhile. Definitely not an item we should address as part of this PR imo. 

What we should do at some point is add a check that runs all the migrations against this mock state and throws an error if the mock state changes at all (i.e. a test that verifies the mock state is already fully migrated/up-to-date)